go.mod

```diff
@@ -0,0 +1,71 @@
+module github.com/armory-io/terraform-provider-spinnaker
```

Comment:
The README should be updated to document how to build this if Go modules are used. Otherwise, it may be better in a separate pull request, since this project already tracks its dependencies using dep.